--- docker-compose.yml	2023-02-07 09:46:54.895930796 +0100
+++ docker-compose.yml.deploy	2023-01-31 21:22:36.420116141 +0100
@@ -5,6 +5,7 @@
     image: nginx:latest
     volumes:
       - ./nginx.conf:/etc/nginx/nginx.conf:ro
+      - ./gitver:/var/www/gitver
     depends_on:
       - api
       - web
@@ -8,12 +8,8 @@
     depends_on:
       - api
       - web
-    ports:
-      - "80:80"
   db:
     image: postgis/postgis:14-3.3  # This version was recommended "for new users" 2022-11-03
-    volumes:
-      - /var/lib/smartakartan_db:/var/lib/postgresql/data
     environment:
       TZ: Europe/Stockholm
       POSTGRES_PASSWORD: nwKnN7kgftitCpc  # TODO: Security, see "Docker secrets" https://hub.docker.com/_/postgres

@@ -21,16 +20,12 @@
   api:
     build: .
     environment:
-      - HOST=localhost
-      - SCHEME=http
+      - HOST=smartakartan.entrop.mywire.org
+      - SCHEME=https
     depends_on:
       - db
-    volumes:
-      - ./:/code
   web:
     build: ./react-frontend
     depends_on:
       - api
-    volumes:
-      - ./react-frontend/src:/react-frontend/src
-      - ./react-frontend/public:/react-frontend/public
+
